src/status_im/ui/screens/chat/message/reactions.cljs

```diff
@@ -93,3 +96,72 @@
              [render message {:modal         true
                               :on-long-press #()
                               :close-modal   on-close}]]])]))))
+
+(defn with-reaction-picker []
```

Comment:
could you please elaborate on this ? is this new function or just moved ?

Reply:
Actually, naming is incorrect. Should be `with-context-drawer` or something like that. The difference is it shows Context Drawer instead of reaction picker. This is only for the New UI.

Reply:
so is it a new function ?

Reply:
Yes, I separated functions from old and new ui version. The new UI version now shows Context Drawer while the old UI version launches an animation on the message and shows reaction picker.